persistence-modules/spring-jpa-3/src/main/java/com/baeldung/jpa/subtypes/SubTypeJPAApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.jpa.subtypes;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication 
+@EnableJpaRepositories("com.baeldung.jpa.subtypes.repository")
+public class SubTypeJPAApplication {
```

Comment:
We follow https://google.github.io/styleguide/javaguide.html for naming conventions
```suggestion
public class SubTypeJpaApplication {
```